apps/wallet-mobile/src/features/Exchange/common/ProviderItem/ProviderItem.tsx

```diff
@@ -0,0 +1,62 @@
+import {useTheme} from '@yoroi/theme'
+import {Exchange} from '@yoroi/types'
+import * as React from 'react'
+import {StyleSheet, Text, TouchableOpacity, View} from 'react-native'
+
+import {Space} from '../../../../components/Space/Space'
+import {ProviderLabel, ProviderLogo} from '../constants'
+import {useStrings} from '../useStrings'
+
+type Props = {
+  provider: Exchange.Provider
+  fee: number
+  icon: React.ReactNode
```

Comment:
I'd say that `icon` doesn't reflect much the intention, if the `icon` is always the same and it's used to show the selected intention -> `isSelected?: boolean`, if it can be different I'd go for `rightAdornment`, since it's a ReactNode.